blob/blob.go

```diff
@@ -522,6 +524,69 @@ type WriterOptions struct {
 	BeforeWrite func(asFunc func(interface{}) bool) error
 }
 
+// FromURLFn is for use by provider implementations.
+// It allows providers to convert a parsed URL from Open to a driver.Bucket.
+// It takes a bucket name and a map of key/value options. The map is guaranteed
+// to be non-nil, and all keys will be in lowercase.
+type FromURLFn func(context.Context, string, map[string]string) (driver.Bucket, error)
```

Comment:
I think `FromURLFunc` would be more idiomatic.

Reply:
Done.